File: C#/SelectionSort.cs
using System;

class SelectionSort
{
    public static void Sort(int[] arr)
    {
        int n = arr.Length;
        for (int i = 0; i < n - 1; i++)
        {
            int minIdx = i;
            for (int j = i + 1; j < n; j++)
                if (arr[j] < arr[minIdx])
                    minIdx = j;

            int temp = arr[minIdx];
            arr[minIdx] = arr[i];
            arr[i] = temp;
        }
    }

    public static void Main()
    {
        int[] arr = { 64, 25, 12, 22, 11 };
        Sort(arr);
        Console.WriteLine(string.Join(" ", arr));
    }
}

File: Kotlin/selectionSort.kt
fun selectionSort(arr: IntArray) {
    val n = arr.size
    for (i in 0 until n - 1) {
        var minIndex = i
        for (j in i + 1 until n) {
            if (arr[j] < arr[minIndex]) {
                minIndex = j
            }
        }
        val temp = arr[minIndex]
        arr[minIndex] = arr[i]
        arr[i] = temp
    }
}

fun main() {
    val array = intArrayOf(64, 25, 12, 22, 11, -1, 0, -5)
    selectionSort(array)
    println("Sorted Array: ${array.joinToString(", ")}")
}
